build/includes/sdk.mk

```diff
@@ -215,3 +215,18 @@ sdk-shell-csharp:
 sdk-publish-csharp: RELEASE_VERSION ?= $(base_version)
 sdk-publish-csharp:
 	$(MAKE) run-sdk-command-csharp COMMAND=publish VERSION=$(RELEASE_VERSION) DOCKER_RUN_ARGS="$(DOCKER_RUN_ARGS) -it"
+
+# SDK shell for rust 
+sdk-shell-rust: 
+	$(MAKE) sdk-shell SDK_FOLDER=rust 
+
+# Publish the Rust SDK to crates.io
+sdk-publish-rust:
```

Comment:
Heh, does this work? If so, that's neat.

Your idea here is definitely better! I was just thinking of doing `make sdk-shell-rust` and then running the commands in the shell.

Another option which might be a bit cleaner - have a look at `make sdk-publish-csharp` - rather than worrying about a shell (which may be handy anyway), it runs a `publish.sh` file (https://github.com/googleforgames/agones/blob/main/build/build-sdk-images/csharp/publish.sh).

I wonder if you could do something similar here: https://github.com/googleforgames/agones/tree/main/build/build-sdk-images/rust 

WDYT? Bit cleaner?

Reply:
got it! Tested without `cargo publish` and the latest code will work as expected I guess. Please share your thoughts..